packages/ui/src/components/Toaster/index.tsx

```diff
@@ -15,53 +15,37 @@ import { Text } from '../Text'
 const PREFIX = '.Toastify'
 const AUTOCLOSE_DELAY = 6000 // Delay to close the toast in ms
 
-const TOAST_ICONS = {
-  warning: 'alert',
-  info: 'information-outline',
-  success: 'checkbox-circle-outline',
-  danger: 'alert',
-} as const
-
 const styles = {
   toast: (theme: Theme) => css`
     border-radius: ${theme.radii.default};
-    box-shadow: ${theme.shadows.dropdown};
-    min-height: 44px;
-    margin-bottom: ${theme.space['2']};
+    min-height: 52px;
+    width: 344px;
+    padding: 16px 16px 16px 16px;
```

Comment:
for the padding you can use:

```suggestion
    padding: ${theme.space['2']};
```